### DISCUSSION TITLE
Publishing releases in container images

### DISCUSSION THREAD

**Discussion by janosmurai** [stlink-org/stlink#1383] [Q&A (resolved)] [answered]
What do you think about providing the released versions in container images? 
It would help to overcome issues like [this one](https://github.com/stlink-org/stlink/issues/1376)
Implementing this would be a one-time effort since it is possible to create a GitHub action that automatically creates the image and publishes it after every release. 

Thanks!

**Comment by Nightwalker-87** [comment, marked as answer]
Hi @janosmurai. I don't see any efforts worth spending here (for this project), as there are relatively few similar issues regarding this topic compared to the amount of supported distributions, which mainly roll out this toolset via their downstream package systems anyway.
The other current issue (#1364) related to compilation on Windows would for example not be addressable by such an approach, as it derives from general build settings during package creation, rooting down to the implementation of the used build tool. It therefore needs to be resolved with the help of related build tool documentation.

**Comment by janosmurai** [comment]
Hi @Nightwalker-87 ,
Yeah, that makes sense. Thanks for your reply! 

I hope you don't mind, but I am sharing [here](https://hub.docker.com/r/axemsolutions/stlink-org) the link to the stlink-org image I created for my project. Maybe it will be helpful for someone someday :)
